barretenberg/cpp/src/barretenberg/benchmark/ivc_bench/ivc.bench.cpp

```diff
@@ -128,12 +129,12 @@ BENCHMARK_DEFINE_F(IvcBench, ECCVM)(benchmark::State& state)
 {
     ClientIVC ivc;
 
-    BB_REPORT_OP_COUNT_IN_BENCH(state);
     // Perform a specified number of iterations of function/kernel accumulation
     perform_ivc_accumulation_rounds(state, ivc);
 
     // Construct and measure eccvm only
     for (auto _ : state) {
+        BB_REPORT_OP_COUNT_IN_BENCH(state);
```

Comment:
what are these for?

Reply:
The counts are global variables, they need to then get into the state counter variables